sdk/common/src/main/java/io/opentelemetry/sdk/internal/ExtendedAttributesMap.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.internal;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.incubator.common.ExtendedAttributeKey;
+import io.opentelemetry.api.incubator.common.ExtendedAttributes;
+import io.opentelemetry.api.incubator.common.ExtendedAttributesBuilder;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiConsumer;
+import javax.annotation.Nullable;
+
+/**
+ * TODO.
```

Comment:
Probably need to flesh out this TODO --- I have no clue what is to be done. 🤣

Reply:
Just a reminder to add javadoc. Will fix!